From 6eeabaa9890ed10983f6ac52070b65189210ea8f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jan=20Jan=C3=9Fen?= <janssen@mpie.de>
Date: Wed, 26 Feb 2025 10:38:24 +0100
Subject: [PATCH] setup

---
 setup.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 8409993..ba3b097 100644
--- a/setup.py
+++ b/setup.py
@@ -1,4 +1,5 @@
 import os
+import sys
 from setuptools import setup, find_packages
 from setuptools.extension import Extension
 
@@ -15,7 +16,7 @@
               Extension("*", ["atomman/defect/*" + EXT])]
 
 if USE_CYTHON:
-    extensions = cythonize(extensions)
+    extensions = cythonize(extensions, language_level=sys.version_info[0], force=True)
 
 def getversion():
     """Fetches version information from VERSION file"""
